tests/sentry/api/endpoints/test_project_stacktrace_link.py

```diff
@@ -51,38 +51,64 @@ def setUp(self):
         self.repo.provider = "example"
         self.repo.save()
 
-        self.config = self.create_code_mapping(
+        self.code_mapping1 = self.create_code_mapping(
```

Comment:
We now have 4 code mappings even though we only do something useful with number 2.